modules/openid_auth/spec/requests/validation_request_spec.rb

```diff
@@ -91,5 +98,25 @@
         expect(JSON.parse(response.body)['errors'].first['code']).to eq '500'
       end
     end
+
+    it 'should return a not found when va profile returns not found', :aggregate_failures do
```

Comment:
I thought you were marking this guy as pending?

Reply:
The change that Patrick made doesn't require this to be pending anymore.